docker/ubuntu_22.04-dev.dockerfile

```diff
@@ -1,22 +1,19 @@
 ARG py_version=3.8
```

Comment:
Is there a reason why Python 3.8 and not 3.10 which is the default on Ubuntu 22.04?

Reply:
Good catch @ngranda - I think getting rid of this build arg entirely probably makes sense